### PR TITLE
Add dropdown hover in nav bar

### PR DESCRIPTION
**Pull request by KristaANelson**
Updated categories and account to dropdown on hover. 

Created "X" for cancel on login form.

### CODE REVIEW

**Comment by KristaANelson** [comment]
Add "X" to create user

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -108,6 +108,9 @@ ul.dropdown-menu p.navbar-text {
   margin-top: 10px;
 }
 
+.navbar-nav > li:hover > .dropdown-menu {
+  display: block;
+}
```
Comment (scottcrawford03) [inline app/assets/stylesheets/application.css.scss:113]: :santa: is happy. looks good.

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -166,3 +169,12 @@ hr {
   font-family: 'Langdon';
   text-transform: uppercase;
 }
+
+.cancel {
```
Comment (scottcrawford03) [inline app/assets/stylesheets/application.css.scss:173]: :x: on lock.

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -1,4 +1,5 @@
 <div class="login-form">
+  <%= link_to "X", root_path, class: "cancel" %>
```
Comment (scottcrawford03) [inline app/views/users/new.html.erb:2]: :earth_africa: awesome stuff

**Pull request merged**